Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -386,7 +386,7 @@ fi
 
 if [ -n "$PYTHON_VENV_PACKAGE" ]; then
     $APT_GET install $PYTHON_VENV_PACKAGE
-    python3 -m venv $HOME/venv-ardupilot
+    python3 -m venv --system-site-packages $HOME/venv-ardupilot
```

Comment:
can you also change the others (eg. arch) so we are as consistent as we can be between OSes

Reply:
Yep, done!